modules/nw-osp-pod-connections-ipv6.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/network-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="nw-osp-pod-connections-ipv6_{context}"]
+= Enabling IPv6 connectivity to pods on {rh-openstack}
+
+To enable IPv6 connectivity between pods that have additional networks that are on different nodes, disable port security for the IPv6 port of the server. Disabling port security obviates the need to create allowed address pairs.
+
+[IMPORTANT]
+====
+Only the following IPv6 additional network configurations are supported:
+
+* SLAAC and host-device
+* SLAAC and MACVLAN
+* DHCP stateless and host-device
+* DHCP stateless and MACVLAN
+====
+
+.Procedure
+
+* On a command line, enter:
++
+[source,terminal]
+----
+$ openstack port set --no-security-group --disable-port-security <compute_ipv6_port>
```

Comment:
I would like to double check this step first

Reply:
@MaysaMacedo How's this going? Would prefer to have this merged by freeze, but can probably buy a day or two if it's important.

Reply:
The command is fine if we explain what it does (remove traffic restriction). It's not really practicable to keep port security knowing that the MAC address and hence IPv6 is generated once the pod starts.